src/context/ExtensionsContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {
+  createContext,
+  FC,
+  PropsWithChildren,
+  useContext,
+  useMemo,
+} from 'react';
+import { useParams } from 'react-router-dom';
+import { useExtensionData } from '~hooks';
```

Comment:
Add an empty line between imports packages and other imports (like hooks, types, components, etc)